pr_agent/git_providers/github_provider.py

```diff
@@ -154,10 +154,20 @@ def get_diff_files(self) -> list[FilePatchInfo]:
     def publish_description(self, pr_title: str, pr_body: str):
         self.pr.edit(title=pr_title, body=pr_body)
 
+    def publish_persistent_review(self, pr_comment: str):
+        prev_comments = list(self.pr.get_issue_comments())
+        for comment in prev_comments:
+            if comment.body.startswith('## PR Analysis'):
+                pr_comment_updated = pr_comment.replace('## PR Analysis\n', '## PR Analysis (updated)\n')
```

Comment:
Just wondering if maybe this is too specific to the use-case of the `/review` command?

For comparison, the `publish_comment` method can be used by all commands.

Maybe this method can be generalized as `publish_persistent_commment` and receive an extra argument - `comment_tag: str`. In the case of the review command, it will be thus called with `comment_tag="## PR Analysis"`.